assets/components/switchable/errorComponents/paymentError.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import { classNameWithModifiers } from 'helpers/utilities';
+
+
+// ----- Types ----- //
+
+type PropTypes = {
+  paymentMethod: string,
+  modifierClass: string,
+};
+
+
+// ----- Component ----- //
+
+function PaymentError(props: PropTypes) {
+  return (
+    <div className={classNameWithModifiers('component-payment-error', [props.modifierClass])}>
+      <p className="component-payment-error__message">
+        We are currently experiencing issues with {props.paymentMethod} payments.
```

Comment:
I'd perhaps say: We are not accepting {props.paymentMethod} payments at the moment.